### ISSUE TITLE
Airshield Projectors (the trader handheld ones) don't block atmos.

### ISSUE THREAD

**Issue by Eneocho** [d3athrow/vgstation13#33346]
#### Description of issue
The airshield projector devices, the ones traders sell, do not work. The airshield effect does not block atmos.
#### Difference between expected and actual behavior
Expected for the n2o flood to be blocked by my handheld Airshield Projector so that I may install a permanent machinery airshield.
The temporary airshields (from the handheld device) did NOT block atmospherics. N2O loose got even worse and medical got flooded not soon after.
#### Steps to reproduce
Flood a room with N2O or Plasma.
Use the temporary airshields from the handheld device to block the atmospherics in the airlock and surrounding tiles, so that you may go through without loosing.
Open the airlock while the temporary airshield is up.
Realize the vox scammed you for 75 credits and that the airshield generator device fields do not shield air.
#### Length of time in which bug has been known to occur
They worked like 2 months ago, dunno what broke them but I blame the code reworks.
[bugfix]

TO MAKE IT VERY, VERY CLEAR: THIS REFERS TO THE HANDHELD DEVICES, NOT THE MACHINERY. 
DO NOT CLOSE ASSUMING IT GOT FIXED BY THE MACHINERY FIXES, THE TEMPORARY FIELDS ARE NOT MACHINERY-BASED BUT RATHER EFFECT-BASED.
This is a duplicate of #33344, since that got closed by mistake.

**Comment by Dacendeth** [comment]
![image](https://user-images.githubusercontent.com/26881007/191628338-57f75e35-cac3-4fd6-b4d1-a621087aa7e1.png)
Skill issue? works on test.

**Comment by Eneocho** [comment]
I've used them the last 2 or 3 rounds I've played and each time the field did NOT block atmos. A detail I forgot to mention is that both times the fields were on vertical airlocks (you go up or down the airlock to go through, not left/right).

**Comment by Kurfursten** [comment]
![image](https://user-images.githubusercontent.com/9782036/191629857-7285ae94-992f-4713-8e8c-506c5b360185.png)

Skill issue?

**Comment by Eneocho** [comment]
I dunno what the fuck is going on then cause they failed every time while the thing still had like 5 seconds before closing.
Guess it IS fixed, but I will reopen this if I see it again.

**Comment by Kurfursten** [comment]
I tried again on two more doors and it held up fine. I guess atmos was just losing it that shift

**Comment by Eneocho** [comment]
The aforementioned shifts were all affected by massive floods, so perhaps you're onto something. I'll have to stress test it locally.
